docs/upgrade/2.12.10_to_2.13.0.rst

```diff
@@ -0,0 +1,80 @@
+
+Upgrade from 2.12.10 to 2.13.0
+==============================
+
+Changes in SecureDrop 2.13.0
+----------------------------
+
+SecureDrop 2.13.10 marks a major change from previous versions as the ``securedrop-admin`` tool is now provided as a Debian package installed on Tails using ``apt`` and available system-wide, rather than as a tool that must be run directly from a local cloned version of the SecureDrop repository. 
```

Comment:
I _think_ local is an adverb here, so it should be:

```suggestion
SecureDrop 2.13.10 marks a major change from previous versions as the ``securedrop-admin`` tool is now provided as a Debian package installed on Tails using ``apt`` and available system-wide, rather than as a tool that must be run directly from a locally cloned version of the SecureDrop repository. 
```

Then again, I don't pretend to have an expert grasp on the English language, so feel free to dismiss if I'm horribly mistaken here.